packs/feats/invoke-the-elements.json

```
@@ -32,7 +32,60 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.TraitBludgeoning",
+                        "predicate": [
+                            "feat:brine-may"
+                        ],
+                        "value": "bludgeoning"
+                    },
+                    {
+                        "label": "PF2E.TraitCold",
+                        "predicate": [
+                            "feat:snow-may"
+                        ],
+                        "value": "cold"
+                    },
+                    {
+                        "label": "PF2E.TraitFire",
+                        "predicate": [
+                            "feat:veil-may"
+                        ],
+                        "value": "fire"
+                    },
+                    {
+                        "label": "PF2E.TraitElectricity",
+                        "predicate": [
+                            "feat:virga-may"
+                        ],
+                        "value": "electricity"
+                    }
+                ],
+                "flag": "changelingType",
+                "key": "ChoiceSet"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.changeling.changelingType",
+                "value": "{item|flags.pf2e.rulesSelections.changelingType}"
+            },
+            {
+                "itemType": "feat",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "feat:invoke-the-elements",
+                    "feat:brine-may"
+                ],
+                "property": "traits",
+                "value": "water"
```

```suggestion
                "itemType": "feat",
                "key": "ItemAlteration",
                "mode": "add",
                "predicate": [
                    "item:slug:invoke-the-elements",
                    "feat:brine-may"
                ],
                "property": "traits",
                "value": "water"
```
This would otherwise add the trait to all feats on the actor